HotelManagerment/com.spring.sample/src/com/spring/sample/controller/AuthorizationsController.java

```diff
@@ -1,22 +1,148 @@
 package com.spring.sample.controller;
 
+import java.util.List;
 import java.util.Locale;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import com.spring.sample.entity.Users;
+import com.spring.sample.entity.Permission;
+import com.spring.sample.service.PermissionService;
+import com.spring.sample.service.UsersService;
 
 @Controller
-@EnableWebMvc
+@RequestMapping("/authorizations")
 public class AuthorizationsController {
 	private static final Logger logger = LoggerFactory.getLogger(AuthorizationsController.class);
 
-	@GetMapping(value = "/authorizations")
+	@Autowired
+	private UsersService usersService;
+
+	@Autowired
+	private PermissionService permissionService;
+
+	@GetMapping
 	public String index(Locale locale, Model model) {
-		logger.info("Requesting roomcategory form");
+		List<Users> listUsers = usersService.getAllUsers();
+		List<Permission> permissions = permissionService.getAllPermissions();
+		model.addAttribute("listUsers", listUsers);
```

Comment:
```suggestion
		model.addAttribute("users", users);
```
đặt tên ri cho nó sang, đã list rồi còn users nữa